src/components/hero-banner/hero-banner.scss

```diff
@@ -82,3 +82,20 @@
   fill: $white;
   display: block !important;
 }
+
+@include media-breakpoint-down(xs) {
+  .hero-title {
+    letter-spacing: 1px;
+    font-size: 3.2rem;
+  }
+  .hero-subtitle {
+    text-align: center;
+    width: 201px;
```

Comment:
not sure if it's possible to use a relative measurement here instead of `px`?  Like `rem` or `%`?

Reply:
Updated changes to use relative measurement.